### PR TITLE
Landing page for the tickets app

### PR DESCRIPTION
**Pull request by julien51**
This is a first implementation of the landing page for the tickets app.

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image


And mobile:

image

Note: we actually do not have a mobile template so I went for something really simple!

**Comment by julien51** [comment]
Note 2: waiting for the SVG logo too!

File: docker/docker-compose.ci.yml [hunk 1]
```diff
@@ -90,6 +90,19 @@ services:
     image: unlock-js
     environment:
       CI: '${CI}'
+  tickets:
```
Comment (julien51) [inline docker/docker-compose.ci.yml]: Hun. hold on, why is this here?

File: tickets/src/components/interface/Layout.js [hunk 2]
```diff
@@ -56,9 +56,18 @@ const Right = styled.div`
...
+  bottom: 0;
+  padding: 100px 0px;
+  display: grid;
+  grid-template-columns: 40px 1fr;
+  align-items: center;
+  height: 50px;
   ${Media.phone`
     display: none;
   `};
+  p {
```
Comment (cellog) [inline tickets/src/components/interface/Layout.js]: ?

**Comment by julien51** [comment]
ok all good! PTAL!

File: tickets/src/components/content/HomeContent.js [hunk 3]
```diff
@@ -1,30 +1,87 @@
...
+    width: 100%;
+  `};
+`
+
+const Grid = styled.section`
+  display: grid;
+  grid-template-columns: 1fr 250px;
+  grid-gap: 60px;
+  ${Media.phone`
+    grid-gap: 0px;
```
Comment (cellog) [inline tickets/src/components/content/HomeContent.js]: nit: save 2 characters by dropping `px`

File: tickets/src/components/content/HomeContent.js [hunk 4]
```diff
@@ -33,6 +90,7 @@ const SubTitle = styled.h2`
   font-weight: 300;
   font-size: 32px;
   line-height: 42px;
+  margin-top: 0px;
```
Comment (cellog) [inline tickets/src/components/content/HomeContent.js]: nit: save 2 characters by dropping `px`

File: tickets/src/components/content/HomeContent.js (hunk 3 above)
Comment (julien51) [reply tickets/src/components/content/HomeContent.js]: ho! good catch!
